benchmark.js

```diff
@@ -734,6 +734,26 @@
       }
     }
 
+    /**
+     * Trigger error for deferred benchmark.
+     *
+     * @memberOf Benchmark.Deferred
+     * @param {Error} [err]
+     */
+    function reject(err) {
+        // make sure error not a void like value
+        var error = err || new Error(err);
+        error.originError = err;
+
+        var deferred = this,
+            clone = deferred.benchmark,
+            bench = clone._original;
+
+        var event = Event('error');
+        clone.error = event.message = error;
```

Comment:
Is it not weird to set the error object as both `error` and `message`? Why not just the `event.error`? Or you could set `error.message` to `event.message`.

Reply:
Yes I agree.